robottelo/hosts.py

```diff
@@ -479,6 +479,118 @@ def install_cockpit(self):
+        setup_insights=False,
+        setup_remote_execution=True,
+        setup_remote_execution_pull=False,
+        lifecycle_environment=None,
+        operating_system=None,
+        packages=None,
+        repo=None,
+        repo_gpg_key_url=None,
+        remote_execution_interface=None,
+        update_packages=False,
+        ignore_subman_errors=True,
+        force=True,
+        insecure=True,
+        username=settings.server.admin_username,
+        password=settings.server.admin_password,
+    ):
+        """Registers content host to the Satellite or Capsule server
+        using a global registration template.
+
+        :param target: Satellite or Capusle hostname to register to, required.
+        :param org: Organization name to register content host for, required.
+        :param loc: Location name to register content host for, required.
+        :param activation_keys: Activation key name to register content host
+            with, required.
+        :param setup_insights: Install and register Insights client, requires OS repo.
+        :param setup_remote_execution: Copy remote execution SSH key.
+        :param setup_remote_execution_pull: Deploy pull provider client on host
+        :param lifecycle_environment: Lifecycle environment.
+        :param operating_system: Operating system.
+        :param packages: A list of packages to install on the host when registered.
+        :param repo: Repository to be added before the registration is performed, supply url.
+        :param repo_gpg_key_url: Public key to verify the package signatures, supply url.
+        :param remote_execution_interface: Identifier of the host interface for remote execution.
+        :param update_packages: Update all packages on the host.
+        :param ignore_subman_errors: Ignore subscription manager errors.
+        :param force: Register the content host even if it's already registered.
+        :param insecure: Don't verify server authenticity.
+        :param username: A user name to register the content host with.
+        :param password: The user password.
+        :return: SSHCommandResult instance filled with the result of the
+            registration.
+        """
+        cmd = 'curl -sS '
+        if username and password:
+            cmd += f'-u {username}:{password} '
+        if insecure:
+            cmd += '--insecure '
+        if target:
+            cmd += f"'https://{target}:9090/register?"
+        else:
+            raise ContentHostError('Target Satellite/Capsule not specified')
+        if activation_keys:
+            cmd += f'activation_keys={activation_keys}'
+        else:
+            raise ContentHostError('Activation key not specified')
+        if org:
+            selected_org = entities.Organization().search(query={'search': f'name="{org}"'})[0]
+            cmd += f'&organization_id={selected_org.id}'
+        else:
+            raise ContentHostError('Organization not specified')
```

Comment:
Is this in case someone overrides the default and passes `None` or `False` for org?  Just remove the default and make it a required parameter.